doc/source/examples/dem/small-scale-rotating-drum/small-scale-rotating-drum.rst

```diff
@@ -0,0 +1,182 @@
+
+.. code-block:: text
+
+    subsection mesh
+        set type                 				= dealii
+        set grid type      	     				= cylinder
+        set grid arguments       				= 0.056:0.051
+        set initial refinement   				= 3
+    end
+
+
+Packing information
+~~~~~~~~~~~~~~~~~~~~
+
+An insertion box is defined inside the cylindrical domain, inserting 8000 particles every 0.5 seconds while the cylinder is at rest.
+
+.. code-block:: text
+
+
+    subsection insertion info
+    	set insertion method				        = non_uniform
+    	set inserted number of particles at each time step      = 8000
+    	set insertion frequency            		 	= 100000
+    	set insertion box minimum x            	 	= -0.05
+    	set insertion box minimum y            	        = 0.
+    	set insertion box minimum z            	        = -0.04
+    	set insertion box maximum x            	        = 0.05
+    	set insertion box maximum y           	 	= 0.04
+    	set insertion box maximum z            	        = 0.04
+    	set insertion distance threshold			= 1.1
+    	set insertion random number range			= 0.05
+    	set insertion random number seed			= 19
+    end
+
+Restart files are written once the packing ends.
+
+Lagrangian physical properties
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The particles are monodispersed with a radius 0.0015 m  and a density of 2500 kg/m3, respectively.
+
+.. code-block:: text
+
+    subsection lagrangian physical properties
+        set gx            		 						= 0.0
+        set gy            		 						= -9.81
+        set gz            		 						= 0.0
+        set number of particle types	                = 1
+            subsection particle type 0
+            set size distribution type					= uniform
+                set diameter            	 			= 0.003
+                set number              				= 20000
+                set density particles  	 				= 2500
+                set young modulus particles         	= 100000000
+                set poisson ratio particles          	= 0.24
+                set restitution coefficient particles	= 0.97
+                set friction coefficient particles      = 0.3
+                set rolling friction particles         	= 0.01
+        end
+        set young modulus wall            				= 100000000
+        set poisson ratio wall            				= 0.24
+        set restitution coefficient wall           		= 0.85
+        set friction coefficient wall         			= 0.35
+        set rolling friction wall         	      	  	= 0.01
+    end
+
+
+Model parameters
+~~~~~~~~~~~~~~~~~
+
+The load balancing method used is the dynamic method, more details are explained in the `Rotating Drum example <../rotating-drum/rotating-drum.html>`_. 
```

Comment:
Here you have more parameters than the load balancing-related ones.

Reply:
If you want to focus on the load balancing part, an extra period explaining it would be nice :)